controllers/velero_test.go

```diff
@@ -3651,12 +3651,12 @@ func TestDPAReconciler_getVeleroImage(t *testing.T) {
 				},
 			},
 			pluginName: common.Velero,
-			wantImage:  "quay.io/konveyor/velero:latest",
+			wantImage:  "quay.io/konveyor/velero:oadp-1.0",
 			setEnvVars: map[string]string{
 				"REGISTRY":    "quay.io",
 				"PROJECT":     "konveyor",
 				"VELERO_REPO": common.Velero,
-				"VELERO_TAG":  "latest",
+				"VELERO_TAG":  "oadp-1.0",
```

Comment:
Shouldn't these blocks (here & in [pkg/credentials/credentials_test.go](https://github.com/openshift/oadp-operator/pull/832/files#diff-0a12f5c252d8d2a8e406ccd3c83025bd31c3b523c18c11e65c73a84d6b4ae26d)) be changed to use `RELATED_IMAGE_` in order to continue to work as expected?

Reply:
I guess so.. the function checks if expected env is defined, else default to values in the code.. which I guess is the same as this test hence it didn't see any differences.